src/en/multporn/src/eu/kanade/tachiyomi/extension/en/multporn/Multporn.kt

```diff
@@ -47,36 +39,22 @@ class Multporn : ParsedHttpSource() {
     // Popular
 
     private fun buildPopularMangaRequest(page: Int, filters: FilterList = FilterList()): Request {
-        val body = FormBody.Builder()
-            .addEncoded("page", page.toString())
-            .addEncoded("view_name", "top")
-            .addEncoded("view_display_id", "page")
+        val url = "$baseUrl/best".toHttpUrlOrNull()!!.newBuilder()
```

Comment:
```suggestion
        val url = "$baseUrl/best".toHttpUrl().newBuilder()
```

Reply:
You need to update the imports too.

This file has several more occurrences of `toHttpUrlOrNull()!!` so you should update those as well.